Tests/YCoreUITests/Protocols/ImageAssetTests.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  ImageAssetTests.swift
+//  YCoreUI
+//
+//  Created by Dev Karan on 21/12/22.
+//  Copyright © 2022 Y Media Labs. All rights reserved.
+//
+
+import XCTest
+import YCoreUI
+
+final class ImageAssetTests: XCTestCase {
+    func test_bundle() {
+        XCTAssertEqual(Flags.bundle, .module)
+        XCTAssertEqual(Icons.bundle, .module)
+    }
+    
+    func test_namespace() {
+        XCTAssertNil(Flags.namespace)
+        XCTAssertEqual(Icons.namespace, "Icons")
+        XCTAssertNil(Missing.namespace)
+    }
+    
+    func test_fallbackImage() {
+        XCTAssertNotNil(Flags.fallbackImage)
+        XCTAssertNotNil(Icons.fallbackImage)
+        XCTAssertEqual(Missing.fallbackImage, UIImage(systemName: "x.squareroot"))
+    }
+    
+    func test_loadImageWithNameSpace() {
+        Icons.allCases.forEach {
+            XCTAssertNotNil($0.loadImage())
+        }
+    }
+    
+    func test_loadImageWithoutNameSpace() {
+        Flags.allCases.forEach {
+            XCTAssertNotNil($0.loadImage())
+        }
+    }
+    
```

Comment:
```suggestion
    
    func test_missingImage() {
        Missing.allCases.forEach {
            XCTAssertNil($0.loadImage())
            XCTAssertEqual($0.image, UIImage(systemName: "x.squareroot"))
        }
    }
```

we're missing a test